kfs-core/src/main/java/edu/arizona/kfs/coa/document/validation/impl/OrgReviewRoleRule.java

```diff
@@ -29,6 +30,51 @@ protected boolean validateFundGroupSubFundGroupCode(OrgReviewRole orr) {
 	}
 	
 	@Override
+    protected boolean validateRoleMember(OrgReviewRole orr){
+        boolean valid = true;
+        if(StringUtils.isNotEmpty(orr.getPrincipalMemberPrincipalName())){
+            if (orr.getPerson() == null) {
+                putFieldError(OrgReviewRole.PRINCIPAL_NAME_FIELD_NAME, "error.document.orgReview.invalidPrincipal"
+                        , getDataDictionaryService().getAttributeLabel(OrgReviewRole.class, OrgReviewRole.PRINCIPAL_NAME_FIELD_NAME) );
+                valid = false;
+            }
+        }
+        if(StringUtils.isNotEmpty(orr.getRoleMemberRoleName())){
+            if ( StringUtils.equals( KFSConstants.SysKimApiConstants.ACCOUNTING_REVIEWER_ROLE_NAME, orr.getRoleMemberRoleName())
+                    || StringUtils.equals( KFSConstants.SysKimApiConstants.ORGANIZATION_REVIEWER_ROLE_NAME, orr.getRoleMemberRoleName() ) ) {
```

Comment:
This method was reworked and has changed a lot from KFS3 to KFS6.  I would assume that you are correct that this should be added since the others are there.  I will add it and retest.